devlib/target.py

```diff
@@ -1688,3 +1688,90 @@ def process_node(node, path, value):
             process_node(result, path, value)
 
     return result
+
+
+class ChromeOsTarget(LinuxTarget):
+
+    os = 'chromeos'
+
+    def __init__(self,
+                 connection_settings=None,
+                 platform=None,
+                 working_directory=None,
+                 executables_directory=None,
+                 android_working_directory=None,
+                 android_executables_directory=None,
+                 connect=True,
+                 modules=None,
+                 load_default_modules=True,
+                 shell_prompt=DEFAULT_SHELL_PROMPT,
+                 package_data_directory="/data/data",
+                 ):
+
+        # Pull out adb connection settings
+        adb_conn_params = ['device', 'adb_server']
+        self.android_connection_settings = {}
+        for setting in adb_conn_params:
+            if connection_settings.get(setting, None):
+                self.android_connection_settings[setting] = connection_settings[setting]
+
+        # If adb device is not explicitly specified use same as ssh host
+        if not connection_settings.get('device', None):
+            self.android_connection_settings['device'] = connection_settings.get('host', None)
+
+        self.android_target = AndroidTarget(connection_settings=self.android_connection_settings,
```

Comment:
I would rename this to just `android` (or maybe `android_container` though that might be a bit long), as `target.android_target` would look weird in the calling code.

Reply:
`android_container` looks better to me... just to be more clear that's an instance of Android running within an host system.